x-pack/plugins/ml/public/file_datavisualizer/components/summary/summary.js

```diff
@@ -35,10 +35,10 @@ function createDisplayItems(results) {
       title: 'Number of lines analyzed',
       description: results.num_lines_analyzed,
     },
-    {
-      title: 'Charset',
-      description: results.charset,
-    }
+    // {
```

Comment:
Is this meant to be commented out? Not sure how it relates to this PR?

Reply:
there is an outstanding issue with the charset of the document being sent to the endpoint always being UTF-8 due to the way it's read by the `FileReader`.
A future enhancement will be to read the file as an ArrayBuffer and then send it to the endpoint as it to preserve the encoding.
For now we should not list the charset because it is inaccurate.